tests/macro/scripts/MPI/gather.py

```diff
@@ -16,17 +16,17 @@
     block_length = nb_values / size
 
     # ...
-    values = [0]*block_length
+    values = [0] * block_length
```

Comment:
I remember emily once said that we should not make norm changes.

Reply:
It's not that you shouldn't make norm changes. It's just that it makes it harder to review a PR when there are things in it that aren't relevant. It gives the impression that this PR changes something in MPI when it doesn't.
It's mostly courtesy. There's no need for all 3 reviewers to double check a file that didn't really change.

If we want to make changes to eg normalise the spacing conventions then we could create a PR for this and add another linter to keep it clean but this isn't a priority as the readability isn't very different and it doesn't affect the execution